docs/bidders/appnexus.md

```diff
@@ -17,3 +17,29 @@ is not supplied for an `imp`, but `request.app.ext.prebid.source`
 and `request.app.ext.prebid.version` are supplied, the adapter will fill in a value for
 `diplaymanagerver`. It will concatonate the two `app` fields as `<source>-<version>` fo fill in
 the empty `displaymanagerver` before sending the request to AppNexus.
+
+## Test Request
+
+The following test parameters can be used to verify that Prebid Server is working propely with the 
+Appnexus adapter. This example includes the `imp` object as the test creative must be valid for
+serving into the test impression.
```

Comment:
I find this sentence a bit confusing. Is it missing a coma?
```
This example includes the `imp` object as the test creative must be valid for serving into the test impression.
```
Or did we mean: 
This example includes the `imp` object as the test creative **and it** must be valid for serving into the test impression.

Reply:
No, your sentence says the imp is the test creative, which is not so. We include the imp object so someone using this as a test can create an imp object that the creative can deliver in to.